ShellScript/ShellScript.py

```diff
@@ -2,12 +2,24 @@
 import os
 import re
 import shlex
+import sys
 import Default.exec
 
 
 class RunShellScriptCommand(Default.exec.ExecCommand):
 
-    def run(self, **kwargs):
+    def run(
+        self,
+        working_dir="",
+        encoding="utf-8",
+        env={},
+        kill=False,
+        kill_previous=False,
+        word_wrap=False,
+        syntax="Packages/Text/Plain text.tmLanguage",
```

Comment:
All the parameters that are equivalent to the default and are not modified or read in the override can be omitted, which afaics should be everything except `working_dir`.

Reply:
I don't agree.

Reply:
The command can theoretically be bound to a key in a sublime-keaymap file to call the command. The explicit presence of those arguments is required for PackageDev to provide proper argument completions.

```
{
	"keys": [""],
	"command": "run_shell_script",
	"args": {
		"working_dir": "",
		"encoding": "utf-8",
		"env": {},
		"kill": false,
		"kill_previous": false,
		"word_wrap": false,
		"syntax": "Packages/Text/Plain text.tmLanguage"
	},
},
```

Reply:
Furthermore not passing those arguments (especially `kill`) to default `ExecCommand` was a primary reason for #3621 as well as `{kill: true}` being missing from sublime-build file.

Reply:
I guess I didn't make it clear that `**kwargs` should still be forwarded. 

PackageDev completions are an argument I understand, but that sounds much more like a PD implementation detail than a reason to make a default package plugin repeat the same arguments with defaults as the `exec` command that it eventually forwards everything to.

Reply:
Somehow didn't have `**kwargs` forwarding being a solution in mind at all.